libs/database-entities/src/main/java/gov/cdc/nbs/entity/odse/PostalEntityLocatorParticipation.java

```diff
@@ -65,13 +71,14 @@ public PostalEntityLocatorParticipation(
     }
 
     public void update(final PatientCommand.UpdateAddress update) {
+
         this.asOfDate = update.asOf();
         this.cd = update.type();
         this.useCd = update.use();
         this.locatorDescTxt = update.comments();
-        this.locator.update(update);
-
         changed(update);
+
+        this.locator.update(update);
```

Comment:
no change here?  or does changing the order matter?

Reply:
no change here, I was separating the method calls into Participation changes first, then Locator changes.  The result is the same.